cran-comments.md

```diff
@@ -1,4 +1,29 @@
-# Version 0.9.2
+# Resubmission
+
+This is a resubmission, responding to feedback from Gregor Seyer on initial submission below. 
+
+Gregor wrote:
+
+> Please do not modify the user's global environment or the user's home
+filespace in your examples or vignettes by deleting objects
+rm(list = ls())
+
+This is now fixed.
+
+> Have the issues why your package was archived been fixed?
+Please explain this in the submission comments.
+
+This package was archived because it was not updated to comply with R 4.0.0
```

Comment:
@nathanielolin - if you want to wordsmith this explanation, that would be very welcome

Reply:
Took a shot (used language from https://cran.r-project.org/doc/manuals/r-devel/NEWS.html and shortened)